internal/vds/vds_test.go

```diff
@@ -909,17 +909,17 @@ func TestSurfaceWindowVerticalBounds(t *testing.T) {
  */
 func TestSurfaceHorizontalBounds(t *testing.T) {
 	fill   := float32(-999.25)
-	xinc   := float64(well_known_grid.xinc)
-	yinc   := float64(well_known_grid.yinc)
-	rot    := float64(well_known_grid.rotation)
+	xinc   := float64(samples10_grid.xinc)
+	yinc   := float64(samples10_grid.yinc)
+	rot    := float64(samples10_grid.rotation)
 	rotrad := rot * math.Pi / 180
 
-	horizon := [][]float32{ { 4, 4 }, { 4, 4 }, { 4, 4 } }
+	horizon := [][]float32{ { 16, 16 }, { 16, 16 }, { 16, 16 } }
 
 	targetAttributes := []string{"samplevalue"}
 	interpolationMethod, _ := GetInterpolationMethod("nearest")
-	const above = float32(0.0)
-	const below = float32(4.0)
+	const above = float32(8.0)
+	const below = float32(8.0)
```

Comment:
We are testing horizontal bounds here. The vertical window is irrelevant to the test as long as it's within bounds. We just need to pick _something_. Like we pick interpolation "nearest", or attribute "samplerate". 

The current choice means we get 5 samples from the vds (in the vertical domain) which will be the minimum when we interpolate ourselves. But it can also be 0, 20. Or anything else